<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1820</width>
    <height>913</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="confederate" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="subject" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>151</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="predictions" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>761</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interlocutor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Participant</string>
    </property>
   </widget>
   <widget class="QWidget" name="animation" native="true">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>50</y>
      <width>821</width>
      <height>741</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Brain Regions</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>20</width>
      <height>811</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>931</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>810</y>
      <width>1821</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Brain Visualization</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>791</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1821</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>10</y>
      <width>20</width>
      <height>811</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>931</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>830</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>830</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>830</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Working directory:</string>
    </property>
   </widget>
   <widget class="QWidget" name="audio" native="true">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>210</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select/Deselect All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>142</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Features</string>
    </property>
   </widget>
   <widget class="QPushButton" name="predictButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>142</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Prediction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="simulationButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>142</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="predictorsButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>142</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Predictors </string>
    </property>
   </widget>
   <widget class="QLabel" name="predLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="generateLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>      </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1820</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
